tests/roles/development_environment/files/pre_launch.bash

```diff
@@ -32,8 +32,12 @@ ${BASH_ALIASES[openstack]} image show cirros || \
     ${BASH_ALIASES[openstack]} image create --container-format bare --disk-format $DISK_FORMAT cirros < /tmp/$RAW
 
 # Create flavor
+HPARGS=" "
+if [ "${EDPM_CONFIGURE_HUGEPAGES:-false}" = "true" ] ; then
+    HPARGS="set huge_pages --property hw:mem_page_size=2MB"
+fi
 ${BASH_ALIASES[openstack]} flavor show m1.small || \
-    ${BASH_ALIASES[openstack]} flavor create --ram 512 --vcpus 1 --disk 1 --ephemeral 1 m1.small
+    ${BASH_ALIASES[openstack]} flavor create $HPARGS --ram 512 --vcpus 1 --disk 1 --ephemeral 1 m1.small
```

Comment:
What about creating another flavor `m1.small-hp` for huge pages (always), and then having the `EDPM_CONFIGURE_HUGEPAGES` variable select the flavor we want to use for the VM? I think it would be a bit nicer to have the flavor properties static, but not sure if that's just my opinion, i can't back it with strong reasoning 😀. So not a blocker, just something for consideration.

Reply:
We can also just merge and refactor in a follow up if the above sounds nicer to you as well.

Reply:
yes, let's refactor it later please